tests/test.js

```diff
@@ -266,6 +266,25 @@ mpmodule("mixpanel.track");
         });
     });
 
+    asyncTest("check no property name aliasing occurs during minify", 1, function() {
```

Comment:
i think we can just test that we can override the token with a new token in the properties object